test/helper_functions.sh

```diff
@@ -104,16 +104,14 @@ function split_forked_file() {
 
 	cp $1 $INPUT
 
-	local N=0
-	local PID="fork"
+	local N=1
+	local PID=$(head -n1 $INPUT | cut -d" " -f2)
 	while true; do
 		NAME="${NAME1}-${N}-${NAME2}"
 		touch $NAME
 		set +e
-		[ "$PID" == "fork" ] \
-			&& grep "$PID" $INPUT | sort -k3 -r > $NAME \
-			|| grep "$PID" $INPUT > $NAME
-		grep -v "$PID" $INPUT > $GREP
+		grep -e "$PID" $INPUT > $NAME
+		grep -v -e "$PID" $INPUT > $GREP
```

Comment:
This seems to be unrelated change. Can you split it to separate commit and describe what and why you do that?